sdk/eventgrid/Microsoft.Azure.EventGrid/src/Microsoft.Azure.EventGrid.csproj

```diff
@@ -2,10 +2,10 @@
   <PropertyGroup>
     <AssemblyTitle>Microsoft Azure EventGrid Library</AssemblyTitle>
     <Description>This library can be used to publish events to Azure Event Grid and to consume events delivered by EventGrid. It also defines the event schemas for the events published to EventGrid by various Azure services. </Description>
-    <VersionPrefix>3.0.0</VersionPrefix>
+    <VersionPrefix>3.2.0</VersionPrefix>
```

Comment:
Why did we skip `3.1.0`?

Reply:
When I checked NuGet, It looks like there's already a 3.1.0 version released (https://www.nuget.org/packages/Microsoft.Azure.EventGrid/3.1.0) 1 month back. However, 13 days back the SDK team moved the data plane files from src/sdks to sdk directory, and it looks like as part of that the version checked in into this CSPROJ reverted back to 3.0.0. Hence, I moved the version to 3.2.0.

Reply:
/cc @weshaggard @chidozieononiwu @kurtzeborn for the issues with the move of the library.

Reply:
As part of the move @chidozieononiwu did in https://github.com/Azure/azure-sdk-for-net/pull/6001 it looks like there were conflicting versions the AssemblyVersion was 3.0.0.0 and the PackageVersion was 3.1.0 and we incorrectly picked 3.0.0 instead of 3.1.0 as we should have. So it is correct to go to 3.2.0 now.